### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by christianotieno**

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+linters:
+stylelint:
+config: ‘stylelint.config.js’
```
Comment (bolah2009) [inline .stickler.yml]: Let's mind the indentation. See this. :point_down: 

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+                    <li><a href="#">Opinion: If Trump Goes Down, He’s Taking Everyone With Him</a></li>
+                    <li><a href="#">What Does Having a Boyfriend Have to Do With Sleep?</a></li>
+                    <li><a href="#">As Impeachment Fights Begin, Administration and Congress Clash Over Deposition</a></li>
+                    <li><a href="#">Finding Balance in a Tiny, Wobbly Boat</a></li>
+                </ul>
+            </div>
+        </div>
+        <footer class="footer-navigation">
+            <div class="top-header">
+                <!-- <div class="border-divider"></div> -->
```
Comment (bolah2009) [inline index.html]: #### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,504 @@
...
+                <figure>
+                    <img src="https://static01.nyt.com/images/2019/09/06/science/chandrayaan-2-vikram-moon-photos-1567705670981/chandrayaan-2-vikram-moon-photos-1567705670981-threeByTwoSmallAt2X-v3.jpg?quality=75&auto=webp&disable=upscale" alt="">
+                    <figcaption>India’s Chandrayaan-2 Marks 60 Years of Moon Crashes and Hard Landings <br /> Sept. 20</figcaption>
+                </figure>
+
+                <figure>
+                    <img src="https://static01.nyt.com/images/2019/09/10/science/10INDIAMOON1/10INDIAMOON1-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="">
+                    <figcaption>Manjunath Kiran/Agence France-Presse — Getty Images. Did India’s Chandrayaan-2 Moon Lander Survive? The Chances Are Slim <br /> Sept. 10</figcaption>
+                </figure>
+                <!-- <h4>Editors' Picks</h4> -->
```
Comment (bolah2009) [inline index.html]: Why are you commenting out this heading? :thinking:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi Team,

The [clone](https://rawcdn.githack.com/RoodzFernando/Positioning-and-Floating-Elements/8fc50c60ce5081f703ebeba7b8faab9b1a229044/index.html) looks good.

  - Set up stickler linter
  - checkout other comments below
  - [NEXT PROJECT]: Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) and Check [this awesome article](https://chris.beams.io/posts/git-commit/)
  - [REQUIRED] Use meaningful names for  Pull Requests.
  - [REQUIRED] Add a short summary of changes in Pull Request description.

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Comment by christianotieno** [comment]
The linter is failing us. It is highlighting errors that are not in existence

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @ChristianOtieno and @RoodzFernando  🌟

**Good work on this project, it looks good so far, but we need to fix the issues below before moving on ⬇️** 

- You guys have to fix the stickler errors, check out the configuration setup on [here](https://github.com/microverseinc/linters-config/tree/master/css)

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

**Comment by RoodzFernando** [comment]
We have I have complied with all the code reviews and our project has met all the changes recommended by the reviewers.

**Review by Redvanisation — APPROVED** [review]
### _Nice work on this project, it seems that you guys have met the requirements so I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Pull request merged**